### DISCUSSION TITLE
Snap turn in seamoth

### DISCUSSION THREAD

**Discussion by bennodxb** [Okabintaro/SubmersedVR#136] [Ideas]
I would love the option for snap turn in the seamoth, preferably in both axes. Its a tough ride for motion sick people driving that thing.

Theres already a mod for snap turn which includes seamoth but it doesnt seem to be compatible with this.

Thanks

**Comment by TimbothyAracnio** [comment]
agreed. pleeeaaaassseeeeee!!!!!!
